man/import_gr_skey_tif_to_pg_rast.Rd

```diff
@@ -27,6 +30,12 @@ import_gr_skey_tif_to_pg_rast(
 \item{pg_conn_param}{Keyring object of Postgres credentials, defaults to dadmtools::get_pg_conn_list()}
 
 \item{dst_tbl}{Destination table of the imported gr_skey_tbl table (format: schema_name.table_name), defaults to "whse.all_bc_gr_skey"}
+
+\item{rast_sch}{Destination schema}
+
+\item{pg_rast_name}{Destination pg raster name}
+
+\item{geom_type}{geomtry type of output pg gr_skey table}
```

Comment:
Please add:
Available options are: Centroid, Polygon (any else?). Defaults to "Centroid"

Reply:
Yes correct and polygon auto becomes multipolygon